docs/markdown/Fs-module.md

```diff
@@ -215,3 +215,22 @@ fs.stem('foo/bar/baz.dll.a')  # baz.dll
    specified by `path` changes, this will trigger Meson to reconfigure the
    project. If the file specified by `path` is a `files()` object it
    cannot refer to a built file.
+
+
+### copyfile
+
+*Since 0.62.0*
+
+Copy a file from the source directory to the build directory
+
+Has the following positional arguments:
+   - src `File | str`: the file to copy
+
+Has the following optional arguments:
+   - dest `str`: the destination in the build dir to copy to. If unset will be the basename of the src
```

Comment:
This is wrong. It should say filename and explicitly state that you can't set the output path and trying to do that is a fatal error.

Reply:
I don't see how this conversation is resolved, it still has that doc issue.

That being said, this is basically a fancy sort of custom_target and we just document the output kwarg as "list of output files", so I do not think we need to explicitly talk about how output paths are wrong.

I *do* think this should say "output file" not "destination".

Reply:
Ping?